samples/QmlArExample/qml/main.qml

```diff
@@ -39,7 +41,44 @@ ApplicationWindow {
         id: sceneView
         anchors.fill: parent
         scene: sceneLoader.item ? sceneLoader.item.scene : null
-        onMousePressed: arcGISArView.setInitialTransformation(mouse.x, mouse.y); // for touch screen events
+
+        // If m_screenToLocationMode is true, create and place a 3D sphere in the scene. Otherwise set the
+        // initial transformation based on the screen position.
+        onMousePressed: {
+            // If "screenToLocation" mode is enabled.
+            if (screenToLocationMode)
+            {
+                // Get the real world location for screen point from AR view.
+                const point = arcGISArView.screenToLocation(mouse.x, mouse.y);
+                if (!point)
+                    return;
+
+                // Get or create graphic overlay
+                var graphicsOverlay = graphicsOverlays.get(0);
+                if (!graphicsOverlay)
+                {
+                  graphicsOverlay = ArcGISRuntimeEnvironment.createObject("GraphicsOverlay");
+                  graphicsOverlays.append(graphicsOverlay);
+                }
+
+                // Create and place a graphic at the real world location.
+                const sphereParams = {
+                    style: Enums.SimpleMarkerSceneSymbolStyleSphere, color: "yellow",
+                    width: 0.2520, height: 0.25, depth: 0.25,
+                    anchorPosition: Enums.SceneSymbolAnchorPositionBottom };
+                const sphere = ArcGISRuntimeEnvironment.createObject("SimpleMarkerSceneSymbol", sphereParams);
+
+                const sphereGraphicParams = { geometry: point, symbol: sphere };
+                const sphereGraphic = ArcGISRuntimeEnvironment.createObject("Graphic", sphereGraphicParams);
+
+                graphicsOverlay.graphics.append(sphereGraphic);
```

Comment:
I like the usage of the params objects in this block. That is a good idea to help clarify the `ArcGISRuntimeEnvironment.createObject` calls.